gdraw/gtextinfo.c

```diff
@@ -367,30 +360,111 @@ return( true );
 return( false );
 }
 
-static void ImageCacheReload(void) {
+static GImage *GadgetNormalizeImageFilenames(const char *inname, char **relname, char **absname) {
+    int l, k;
+    char *home = getenv("HOME"), *path;
```

Comment:
That's a pretty big assumption that this is set. The gutils/fsys functions exist for a reason.

Reply:
It's not obvious but that heuristic is just moved from a different place in the current master. Can you give me a slightly more specific pointer to the proper method?

Reply:
I'm thinking of getUserHomeDir https://github.com/fontforge/fontforge/blob/master/gutils/fsys.c#L909

Reply:
Done